pages/code-review.md

```diff
@@ -14,28 +14,61 @@ channel!
 
 ## Why reviews?
 
-Code reviews are incredibly important step to the development process. Our goals
-when reviewing code are to:
+Code reviews are incredibly important step to the development process. Our
+goals when reviewing code are to:
 
 - expose bugs before they make it to production;
 - ensure a consistent quality in the codebase;
 - create an environment for sharing knowledge and developing skills;
+- cross-pollination of debugging skills when problems arise;
+- cultivation of the ability to critique one’s own code more strongly;
 - encourage open communication between the entire team.
 
+## Potential obstacles to be mindful of
+
+If you are attempting to establish code reviews as a part of your team's or
+organization's workflow, there are several obstacles you might run into:
+
+- Code reviews require intense concentration when they are to be done right,
+  which in turn requires additional time that will have an impact on story
+  effort estimates in a sprint.
+- Introducing the idea of writing tests and requiring them in order for a code
+  review to pass will take time and commitment from all team members if they
+  have not already bought into the benefits of TDD.
+- Additional training may be needed for folks who are unfamiliar with
+  what linters and/or testing harnesses.
+- Writing a style guide that is firm but reasonable requires time and a strong
+  commitment from all team members to adhere to and hold each other accountable
+  for.
```

Comment:
I personally don't see this as an obstacle, and don't think we would need to write a style guide from scratch. Ruby and Python (and probably other languages) have established community-based style guides that we can adopt and tweak to our needs if necessary. There is also an effort underway to recommend the use of such style guides and automated checks via services like Hound, which reduces the burden on reviewers and helps team members adhere to the style guide.